examples/config.hocon.sample

```diff
@@ -19,24 +19,28 @@
 # "stdin" for reading unencoded tab-separated events from stdin
 # If set to "stdin", JSON documents will not be sent to Elasticsearch
 # but will be written to stdout.
+# "NSQ" for reading unencoded tab-separated events from NSQ
 source = {{elasticsearchInputType}}
 
 # Where to write good and bad records
 sink {
   # Sinks currently supported are:
   # "elasticsearch" for writing good records to Elasticsearch
   # "stdout" for writing good records to stdout
+  # "NSQ" for writing good records to NSQ  
   good: {{elasticsearchGoodOutputDestination}}
 
   # Sinks currently supported are:
   # "kinesis" for writing bad records to Kinesis
   # "stderr" for writing bad records to stderr
+  # "NSQ" for writing bad records to NSQ    
   # "none" for ignoring bad records
   bad: {{elasticsearchBadOutputDestination}}
 }
 
 # "good" for a stream of successfully enriched events
 # "bad" for a stream of bad events
+# "PlainJson" for writing plain json
```

Comment:
I think we need to rename this to `plain-json` to be coherent with the other options